modules/assim.batch/R/pda.emulator.R

```diff
@@ -55,10 +64,10 @@ pda.emulator <- function(settings, params.id = NULL, param.names = NULL, prior.i
     con <- NULL
   }
 
-  bety <- PEcAn.visualization::betyConnect("~/pecan/web/config.php")
+  bety <- betyConnect("~/pecan/web/config.php")
```

Comment:
yeah, I'm still not used to the new connection.

@bcow so I just do
```
bety <- src_postgres(dbname = settings$database$bety$dbname, 
               host = settings$database$bety$host, 
               user = settings$database$bety$user, 
               password = settings$database$bety$password)
```
and don't actually use `betyConnect` function? and when I pass this `bety` to the `query.format.vars` it will work?